server/src/main/scala/io/delta/standalone/internal/DeltaSharingHistoryManager.scala

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (2021) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Putting these classes in this package to access Delta Standalone internal APIs
+package io.delta.standalone.internal
+
+import io.delta.standalone.internal.actions.CommitMarker
+import io.delta.standalone.internal.util.FileNames
+import io.delta.standalone.storage.LogStore
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.Path
+import scala.collection.JavaConverters._
+
+object DeltaSharingHistoryManager {
+  /**
+   * DeltaHistoryManager.getCommits is not a public method, so we need to make local copies here.
```

Comment:
Should we make getCommitsSafe from DeltaHistoryManager public?

Reply:
updated.